tools/c7n_azure/tests_azure/templates/search.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "variables": {
+        "suffix":"[substring(subscription().subscriptionId, 24, 12)]",
+        "name": "[concat('cctestsearch', variables('suffix'))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Search/searchServices",
+            "name": "[variables('name')]",
+            "apiVersion": "2015-08-19",
+            "location": "Central US",
```

Comment:
Can we use the RG location here or are there very limited regions where this can be deployed?

Reply:
widely available - I must have looked at a limited-region service as a reference point. Will fix!